includes/rcon.sh

```diff
@@ -1,57 +1,56 @@
 # shellcheck disable=SC2148
 
+function get_time() {
+    date '+[%H:%M:%S]'
+}
+
 function save_and_shutdown_server() {
-    rconcli 'broadcast Server-shutdown-was-requested-init-saving'
-    rconcli 'save'
-    rconcli 'broadcast Done-saving-server-shuts-down-now'
+    rconcli broadcast "$(get_time) Server shutdown requested. Saving..."
+    rconcli save
+    rconcli broadcast "$(get_time) Saving done. Server shutting down..."
 }
 
 function broadcast_automatic_restart() {
-    time=$(date '+%H:%M:%S')
+    time=
```

Comment:
This variable is empty and not used anymore

Reply:
Thanks, removed the line.